pegasus/migrations/79_migrate_properties_to_dynamodb.rb

```diff
@@ -0,0 +1,22 @@
+require 'cdo/hip_chat'
+require 'properties_api'
+
+Sequel.migration do
+  up do
+    HipChat.log 'Copying <b>>key value pairs</b> to DynamoDB...'
+
+    # There are currently < 1000 rows in production. Don't bother batching.
+    rows = from(:app_properties)
+    rows.each do |row|
+      value = PropertyBag.parse_value(row[:value])
+      property_bag = DynamoPropertyBag.new(row[:app_id], row[:storage_id])
```

Comment:
Yes. `DynamoPropertyBag.set` ends up being a "put" to the DynamoDB API: https://github.com/code-dot-org/code-dot-org/blob/staging/shared/middleware/helpers/property_bag.rb#L106